src/NewTopicSuggestionForAdClusterForm.js

```diff
@@ -0,0 +1,115 @@
+import React, { useState } from 'react';
+import Form from 'react-bootstrap/Form';
+import Button from "react-bootstrap/Button";
+import Col from 'react-bootstrap/Col';
+import Row from 'react-bootstrap/Row';
+import Toast from 'react-bootstrap/Toast';
+import axios from "axios";
+
+const NewTopicSuggestionForAdClusterForm = params => {
+	var labelName = "";
+	const [validated, setValidated] = useState(false);
+	const [show, setShow] = useState(false);
+	const [message, setMessage] = useState("Test message");
```

Comment:
should this be an empty string for prod?

Reply:
Yes. I've made the change.